modules/mauth-authenticator-akka-http/src/test/scala/com/mdsol/mauth/akka/http/MAuthDirectivesSpec.scala

```diff
@@ -191,31 +206,16 @@ class MAuthDirectivesSpec extends WordSpec with Matchers with ScalatestRouteTest
     "reject with a MissingHeaderRejection if header is missing" in {
       Get() ~> route ~> check {
         inside(rejection) {
-          case MissingHeaderRejection(headerName) ⇒ headerName.replaceAll("_", "-").toLowerCase shouldEqual `X-MWS-Authentication`.name
+          case MissingHeaderRejection(headerName) => headerName.replaceAll("_", "-").toLowerCase shouldEqual `X-MWS-Authentication`.name
         }
       }
     }
   }
 
-  "authenticate for V2" should {
+  "authenticate when v2OnlyAuthenticate = true" should {
     lazy val route: Route = authenticate(executor, authenticatorV2, timeout, requestValidationTimeout).apply(complete(HttpResponse()))
     val publicKey = MAuthKeysHelper.getPublicKeyFromString(FixturesLoader.getPublicKey)
 
-    "pass successfully authenticated request with parameters" in {
-      (client.getPublicKey _).expects(appUuid).returns(Future(Some(publicKey)))
-      //noinspection ConvertibleToMethodValue
-      (mockEpochTimeProvider.inSeconds _: () => Long).expects().returns(timeHeader)
-
-      Get("/").withHeaders(
-        RawHeader(MAuthRequest.MCC_TIME_HEADER_NAME, timeHeader.toString),
-        RawHeader(MAuthRequest.MCC_AUTHENTICATION_HEADER_NAME, authHeaderV2),
-        RawHeader(MAuthRequest.X_MWS_TIME_HEADER_NAME, timeHeaderV2.toString),
-        RawHeader(MAuthRequest.X_MWS_AUTHENTICATION_HEADER_NAME, authHeader)
-      ) ~> route ~> check {
-        status shouldEqual StatusCodes.OK
-      }
-    }
-
```

Comment:
Why was this test removed? This test should still be successful no?

Reply:
I moved the test into "authenticate" and added it back for "authenticate when v2OnlyAuthenticate = true". the both should be be successful.  https://github.com/mdsol/mauth-jvm-clients/pull/92/commits/07fa8b14f1df3b53cbccf169c0db9217561879bf